trunner/device.py

```diff
@@ -39,12 +40,44 @@ def wait_for_dev(port, timeout=0):
 
     # naive wait for dev
     while not os.path.exists(port):
-        time.sleep(0.5)
-        asleep += 0.5
+        time.sleep(0.01)
+        asleep += 0.01
         if timeout and asleep >= timeout:
             raise TimeoutError
 
 
+def switch_rpi_ports(state):
+    if state.lower() == 'on':
+        bool_state = 1
+    elif state.lower() == 'off':
+        bool_state = 0
+    else:
+        print('state argument should be "on" or "off"')
```

Comment:
if the above changes would be introduced no "third" value is possible. Even so - I would throw an error here as it's clearly a programmer error.